src/core/Akka/Actor/Inbox.Actor.cs

```diff
@@ -5,69 +5,69 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Akka.Event;
+using System; 
+using System.Collections.Generic; 
+using System.Linq; 
+using Akka.Event; 
 
-namespace Akka.Actor
+namespace Akka.Actor 
 {
     /// <summary>
     /// TBD
     /// </summary>
-    internal class InboxActor : ActorBase
+    internal class InboxActor:ActorBase 
     {
-        private readonly InboxQueue<object> _messages = new InboxQueue<object>();
-        private readonly InboxQueue<IQuery> _clients = new InboxQueue<IQuery>();
+        private readonly InboxQueue < object > _messages = new InboxQueue < object > (); 
```

Comment:
Something is wrong with your auto formatting rules, we don't want to have a space in the generic type parameter ;)

Reply:
Will do.